modules/oc-mirror-about.adoc

```diff
@@ -17,6 +17,8 @@ When using the oc-mirror plug-in, you specify which content to mirror in an imag
 
 The first time you run the oc-mirror plug-in, it populates your mirror registry with the required content to perform your disconnected cluster installation. In order for your disconnected cluster to continue receiving updates, you must keep your mirror registry updated. To update your mirror registry, you run the oc-mirror plug-in using the same configuration as the first time you ran it. The oc-mirror plug-in references the metadata from the storage backend and only downloads what has been released since the last time you ran the tool. This provides update paths for {product-title} and Operators and performs dependency resolution as required.
 
+// TODO: Need to update this description at all, to describe the changes between TP and GA?
```

Comment:
@jpower432 Does this whole file's description of oc-mirror still hold up for 4.11? I wasn't sure if there was anything here that needed to be updated based on any changes that came with 4.11.

Reply:
@bergerhoffer This is still an accurate description of the `heads-only` functionality for `oc-mirror`.